### PR TITLE
:truck: Rename service.py to app.py + change run.sh script

### PR DESCRIPTION
**Pull request by evans-g-crsj**

### CODE REVIEW

File: src/copy_member_to_members.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 """
 from os import environ
 
-import mappings
-from service import build_es_client
+from src import mappings
```
Comment (evans-g-crsj) [inline src/copy_member_to_members.py]: Maybe these imports can be improved. If you have a suggestion, please let me know.

Comment (devbyaccident) [reply src/copy_member_to_members.py]: That should be fine, but you know more Python than I do right now. One question, did you mean to remove this line?
```python
from service import build_es_client
```

**Review by devbyaccident — CHANGES_REQUESTED** [review]
Can you put any new scripts under `scripts/`? That'll keep the repo a little cleaner. Otherwise, just one question about imports and the rest looks good!

File: src/copy_member_to_members.py (hunk 1 above)
Comment (evans-g-crsj) [reply src/copy_member_to_members.py]: No, I meant not to use two lines when importing from the same package
- ` from src import this`
- `from src import that`

I will squeeze the imports in one import.

**Review by devbyaccident — APPROVED** [review]

**Pull request merged**